apps/reaper/mix.exs

```diff
@@ -84,7 +84,7 @@ defmodule Reaper.MixProject do
       {:checkov, "~> 1.0", only: [:test, :integration]},
       {:credo, "~> 1.0", only: [:dev, :test, :integration], runtime: false},
       {:dialyxir, "~> 1.0.0-rc.6", only: :dev, runtime: false},
-      {:divo, "~> 1.1", only: [:dev, :integration], override: true},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in reaper, reaper needs a new mix version